ios/CentralManager.swift

```diff
@@ -36,6 +36,19 @@ class CentralManager: NSObject {
     )
   }
 
+  func stopCentral() {
```

Comment:
Does iOS not provide any utility for this? This, same as Android, just clears some vars from memory, but it does not seem to register it anywhere else. 

For example: https://stackoverflow.com/questions/33263483/disconnect-a-ble-peripheral-in-swift

Reply:
ah I see - yeah i lack some more in-depth knowledge exposure here. thanks a bunch. Really good to have you for feedback

Reply:
ah I see - yeah i lack some more in-depth knowledge exposure here. thanks a bunch. Really good to have you for feedback